geomesa-feature/pom.xml

```diff
@@ -34,6 +34,11 @@
 
         <!-- Test dependencies -->
         <dependency>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo</artifactId>
+            <version>3.0.0</version>
```

Comment:
Good point, I need to double check this... it is used already by scalding I know.